app/src/main/java/com/iic/mokojin/PlayerQueueFragment.java

```diff
@@ -119,6 +125,25 @@ public void onLoaded(List<QueueItem> queueItems, Exception e) {
                     mNonDismissedPositions = new ArrayList(Arrays.asList(positionsArray));
                 }
             });
+            
+            scheduleUpdateClock();
+        }
+
+
+        private void scheduleUpdateClock(){
```

Comment:
I think this should be in the activity, and it should do 
`mAdapter.notifyDataSetChanged()`.
This way you don't need to pass the activity to the adapter.